settings.py

```diff
@@ -234,6 +234,7 @@
     'django_user_agents',
     'logentry_admin',
     'oauth2_provider',
+    'rest_framework.authtoken',
```

Comment:
Is this used?

Reply:
Custom auth class is inheriting from `TokenAuthentication` and overrides `authenticate` method. This is then called by the framework as part of decorator `@authentication_classes((ABDMUserAuthentication,))` in the views.

Reply:
`TokenAuthentication` is part of the base `rest_framework` library so if that's all you are using then you don't need this.